### PR TITLE
Refactor tox.ini again

### PR DESCRIPTION
**Pull request by untitaker**
- Fix unused "test" env: It should be "normal".
- Don't install useless deps in stylecheck
- Actually install flake8 in stylecheck

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-    pyopenssl
-    greenlet
-    pytest
-    pytest-xprocess
-    redis
-    requests
-    watchdog
-
-    uwsgi: uwsgi
+    normal: pyopenssl
```
Comment (untitaker) [inline tox.ini]: @RonnyPfannschmidt Is there a nicer way to express this?

Comment (RonnyPfannschmidt) [inline tox.ini]: not sure off-hand - on pytest itself we generally use a setup with a completely differently named testenv for linting - but we dont lint for all python versions as well, will need a bit to investigate

Comment (untitaker) [inline tox.ini]: I just want to spare the redundant `normal: `-prefixes. I'd like to do something like this:

    [testenv:normal]
    deps=
        pyopenssl
        ....

Unfortunately this doesn't work.

**Comment by untitaker** [comment]
I'll just merge this, ideas still welcome.

**Pull request merged**